### PR TITLE
Review rough voting

### PR DESCRIPTION
**Pull request by Raemon**
This is the first step in seconding up a Nomination Voting Phase for the review. Creates a ReviewVotingWidget which appears both on PostPages for the relevant posts, and in post menu-items.

image

image

### CODE REVIEW

**Comment by jpaddison3** [comment]
Fix lint

**Comment by jpaddison3** [comment]
I do kinda wish the vote UI was visible on PostItems. I dunno, still not that sure how to do it and we might want to just get a move on, but maybe worth discussing.

File: packages/lesswrong/components/review/NominatePostMenuItem.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/NominatePostMenuItem.tsx:86]: My instinct is that even when rushing, we should clean this up before merging to devel.

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:214]: My recommendation is to make these PascalCase at the same time. (`F2`)

File: packages/lesswrong/components/review/ReviewVotingWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingWidget.tsx:84]: This is the lint error.

File: packages/lesswrong/components/review/ReviewVotingWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingWidget.tsx:72]:
```suggestion
```

File: packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx:32]: I'd remove this line, I think it looks better with less whitespace above.

File: packages/lesswrong/components/review/NominatePostMenuItem.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/NominatePostMenuItem.tsx:88]: Can we stick a bit of whitespace above this divider?

File: packages/lesswrong/components/review/ReviewVotingWidget.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingWidget.tsx:71]: I don't love the way this font interacts with the rest of the UI, which is more grey. Feels a bit unstyled, which it is.

**Review by jpaddison3 — APPROVED** [review]
Fix lint, otherwise Approved Subject To Looking At Comments*

\* CEA calls this an ASTLAC, it's up to you to look at comments and do what you think is best, then merge without another review.

File: packages/lesswrong/components/review/NominatePostMenuItem.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/NominatePostMenuItem.tsx]: Actually I solved this the other way – I don't actually want to get rid of NominatePostMenuItem yet because we might still decide all this crazy newfangled Pre-Voting is a bad idea and want to revert to it. Instead just moved the new component elsewhere.

**Comment by Raemon** [comment]
Styling and phrasing notes: I realized after reading JP's comments that this wants to be styled fairly differently in the MenuItem vs PostPage version. I think rather than figure that out right now, I'm going to come back to it after making more progress on other essential features for the Review.

File: packages/lesswrong/components/review/ReviewVotingPage.tsx
Comment (Raemon) [reply packages/lesswrong/components/review/ReviewVotingPage.tsx:214]: I will get to this when dealing with the rest of the voting UI

**Pull request merged**
